webapp/app/home/templates/index.html

```diff
@@ -212,24 +214,86 @@ <h2>Performance  <small> </small></h2>
+                      <canvas id="vpd_roundchart12"></canvas>
+                      <p style="font-weight:normal">daily</p>
+                    </th>
+                    <th width=25% bgcolor="#e7e7e7">
+                      <canvas id="vpd_roundchart0" ></canvas>
+                      <p style="font-weight:normal">weekly</p>
+                    </th>
+                    <th width=15% valign="top" bgcolor="#e7e7e7">
+                      <canvas id="vpd_roundchart10" ></canvas>
+                      <p style="font-weight:normal">daily</p>
+                    </th>
+                    <th width=25% bgcolor="#f3f3f3">
+                      <canvas id="vpd_roundchart3" ></canvas>
+                      <p style="font-weight:normal">weekly</p>
+                    </th>
+                    <th width=15% valign="top" bgcolor="#f3f3f3">
+                      <canvas id="vpd_roundchart13" ></canvas>
+                      <p style="font-weight:normal">daily</p>
+                    </th>
+                    <th width=25% bgcolor="#e7e7e7">
+                      <canvas id="vpd_roundchart4" ></canvas>
+                      <p style="font-weight:normal">weekly</p>
+                    </th>
+                    <th width=15% valign="top" bgcolor="#e7e7e7">
+                      <canvas id="vpd_roundchart14" ></canvas>
+                      <p style="font-weight:normal">daily</p>
+                    </th>
+                  </tr>
+
+
                 </table>
                 <table>
                   <tr >
 
                     <div>
-                      Temperature: 
-                      <span style= "color: rgb(39, 39, 97); font-size: 20px" >&#9632; </span> freeze 
+                      Temperature:
+                      <span style= "color: rgb(39, 39, 97); font-size: 20px" >&#9632; </span> freeze
                       <span style= "color: rgb(6, 163, 85); font-size: 20px" >&#9632; </span> cold
                       <span style= "color: rgb(124, 236, 79); font-size: 20px" >&#9632; </span> warm
                       <span style= "color: rgb(207, 0, 52); font-size: 20px" >&#9632; </span> hot
                     </div>
                     <div>
-                      Humidity: 
-                      <span style= "color: rgb(39, 39, 97); font-size: 20px" >&#9632; </span> dry 
+                      Humidity:
+                      <span style= "color: rgb(39, 39, 97); font-size: 20px" >&#9632; </span> dry
                       <span style= "color: rgb(16, 72, 155); font-size: 20px" >&#9632; </span> low RH
                       <span style= "color: rgb(81, 163, 230); font-size: 20px" >&#9632; </span> high RH
                       <span style= "color: rgb(134, 217, 255); font-size: 20px" >&#9632; </span> wet
                     </div>
+                    <div>
+                      VPD:
+                      <span style= "color: rgb(39, 39, 97); font-size: 20px" >&#9632; </span> dry
```

Comment:
Maybe rather than "dry" and "wet" here we have "very low VPD" and "very high VPD"?

Reply:
Done